src/components/withPersonalDetails.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import getComponentDisplayName from '../libs/getComponentDisplayName';
+import ONYXKEYS from '../ONYXKEYS';
+
+const personalDetailsPropTypes = PropTypes.shape({
```

Comment:
Should this also have the `localCurrencyCode`? I think it is then passed to the `currentUserPersonalDetails` and expecting the `localCurrencyCode`, but it should be defined here already, right?

Reply:
+1

Reply:
+1

Reply:
Added this in the prop type.